bambou/session.go

```diff
@@ -182,9 +202,12 @@ func (s *Session) send(request *http.Request, info *FetchingInfo) (*http.Respons
 	response, err := s.client.Do(request)
 
 	if err != nil {
-		return response, NewError(ErrorCodeSessionCannotProcessRequest, err.Error())
+		return response, NewBambouError("", err.Error())
```

Comment:
Perhaps we need to add a error title? For instance:  'Unable to process request'